### PR TITLE
Add -h command line argument

### PR DESCRIPTION
**Pull request by maboiteaspam**
It displays help from the Usage section of
the README file.
If stdout is detected as piped, it will display
raw MD content.

### CODE REVIEW

**Comment by maboiteaspam** [comment]
code style fix as i could not fix the git history.

File: changelog-maker.js [hunk 1]
```diff
@@ -51,6 +58,16 @@ function replace (s, m) {
   return s
 }
 
+function showUsage (removeMd) {
```
Comment (rvagg) [inline changelog-maker.js]: How about we remove this argument since it's only ever going to come from `process.stdout.isTTY`

**Comment by rvagg** [comment]
lgtm sans comment above. this doesn't feel optimal but it's a step at least

**Comment by maboiteaspam** [comment]
what do you mean by not optimal ?

File: changelog-maker.js (hunk 1 above)
Comment (maboiteaspam) [inline changelog-maker.js]: yes, just help to give it sense and meaning. I can do that later on, i m not short of one or two more pr on that topic ahah :S

**Comment by rvagg** [comment]
ignore my comment, I have nothing concrete to criticise

**Comment by maboiteaspam** [comment]
up to you, this little change has already been chance for lots of experiments and coding (3 new modules as consequence of  this work)! 
Some more won t make a big change imho.

Now, at that very moment, i d prefer go for experimenting in setting up tests and playground for the module (Still did not accepted my epic fail on the MD formatting, i guess).

**Pull request closed without merging**